### ISSUE TITLE
Create personas: partners

### ISSUE THREAD

**Issue by sacamp** [hackforla/website#2705]
### Overview
We need to have personas so that we can evaluate if the features we are developing service the target audience.

### Action Items

- [x] Review this summary of [user groups](https://docs.google.com/document/d/18FnLUCApXPzMB_hdtMmNrLwPcC0dsStwT___IsHWgcU/edit)
- [x] Create an interview guide containing research objectives and questions, and interview questions
- [x] Peer review interviews guides
- [x] Submit interview guide to UXR Lead for review
- [x] Develop scripts for recruitment email, follow up email
- [x] Recruit participants and schedule interviews
- [x] Conduct user interviews with participants
- [x] Conduct affinity mapping in order to distill research of target groups and identify overarching themes
- [x] Create persona from research findings

### Resources/Instructions
- [Resource on writing effective UX interview guides](https://www.nngroup.com/articles/interview-guide/)
- [example research plan and interview guide](https://docs.google.com/document/d/1ELtk7H_uubatggRcTO9imHzdheeiPWkDLs3KZepMwh0/edit) 
- [tutorial on how to add email addresses to the filter](https://github.com/hackforla/product-management/wiki/Forward-emails-to-team-members)
- [User Persona Guide](https://uxplanet.org/guide-to-creating-user-personas-c62586bb7f0a)

### Created Assets
- [Shared Folder with all Partner Persona project documents](https://drive.google.com/drive/folders/1Z-XiA8uf4q9JiHoIfXgYSdg18dzuguBe)
- [Research Plan](https://docs.google.com/document/d/1obc3S5mbcDMcqdlFYkQ9dIZz47bgAZC-NYhGeJ5RE6Q/edit)
- [Interview template](https://docs.google.com/document/d/1vU4CIbDQfnHV7TtWfTj87tXUyIGH810gLh0hNY1SPKg/edit)
- [Email Recruitment Script](https://docs.google.com/document/d/1oxeB15PexGHO3ctMh39vEPu9ZtB_APy8tky5xyuSYQg/edit)
- [Email Reminder Script](https://docs.google.com/document/d/1JThKVO7DmXZ94b-HJ2Ic2mEoiHvBMmKTrptjHo6CYO4/edit)
- [Second Email Reminder Script](https://docs.google.com/document/d/1Y6cab8V7bB7u6gq2gAkOWng_-fCkoE6EOu7sjwkKwig/edit)
- [Spreadsheet with Participant Contact Info](https://docs.google.com/spreadsheets/d/194zmwigD49iNgMKi36oDCFtz9AMcu7MuLnbQk4gB8nM/edit#gid=0)
- [Agenda for request for Bonnie's help obtaining contact info for interview participants](https://github.com/hackforla/website/issues/3565)
- [Affinity Mapping](https://docs.google.com/document/d/1mMv2VO877ZjHQIICGtMuSD_dC07d_W9dV0p9IQKcq2I/edit)
- [Partner Persona](https://docs.google.com/document/d/1u4ok8nwfHMDPMtWNruZ-mHLn2tBWqjfKA2d4K0Y4sXA/edit)
- [Research Recommendations](https://docs.google.com/document/d/14Y0tAiKrWfHPBSSZwr9pWM2cbFnvf1vfgLqPUx4bGdc/edit)
- [Research Presentation](https://docs.google.com/presentation/d/1b4VXYcDgIno90wPNw6vTv2qj-mRDW_0lfCDiHX73WCM/edit#slide=id.g178e899cf39_0_37)

**Comment by alice-milmac** [comment]
Progress: Self-assigned the issue
Blockers: Need to get access to "user groups" document linked above under "Action Items"
Availability: 4+ hours

**Comment by alice-milmac** [comment]
Progress: Review summary of user groups
Blockers: None
Availability: 4 hours 
ETA: Draft of Interview Guide complete by 7/20

**Comment by ExperimentsInHonesty** [comment]
Existing partners
LADOT - Los Angeles Department of Transportation
LACP - Los Angeles, 
DONE


Perspective Partners
GitHub
[Wins Apps](https://oclawin.org/about-ocla/contact-us/)

Future prospective partners
Ruby for Good
Codesmith
General Assembly
other bootcamps

**Comment by alice-milmac** [comment]
Progress: Based on conversation with Bonnie at HfLA website meeting on 07-24-2022, for this project partners will be defined as existing governmental and non-profits partners. In the future we should consider conducting additional interviews with perspective and future partners but now we are prioritizing building those relationships (i.e. don't want to tax the nascent relationship with an interview request). 

New Additions: I added a few more action items to detail all of the steps needed for this project including peer review of template and develop of recruitment materials 

Blockers: None

Availability: 6 hours

ETA: Request peer review of interview guide during first week of August

**Comment by alice-milmac** [comment]
Progress: Updated Research Plan and semi-structured interview template for partner interviews.

Blockers: None

Availability: 10 hours

ETA: Peer review of interview template at UXR meeting on 08-10-2022

**Comment by alice-milmac** [comment]
Progress: Peer reviewed interview template during weekly UXR meeting. Edited to address comments and considerations. Need to get template for email recruitment, possible follow-up message, consent for interview and to record interview, HfLA email address and Calendly log-in credentials. 

Blockers: None

Availability: 6 hours

ETA: Request PM review of interview template at HfLA meeting on 08-16-2022

**Comment by vaisali89** [comment]
@alice-milmac Alice this is the [document](https://docs.google.com/document/d/1WeY1pWjlaazO754TPw7JWRjo_t4Em2Tv2nPlue1TIBw/edit) with the consent form wording as you asked. Hoping this will be helpful

**Comment by alice-milmac** [comment]
Progress: PM reviewed and approved template and research plan during Sunday's website meeting on 08-14-2022. We discussed the process for contacting participants and decided that Bonnie would email partners because of their established relationship. Her email will include recruitment text and method for the partner to schedule their interview. I will conduct each interview provided the participant agrees to being recorded. If not, we will reschedule so that a second researcher can be present to take notes.

Blockers: Need more specifics about recruitment email text and also need to work through logistics of scheduling interviews (i.e. how my availability will be listed in the calendar, how I will get notified about scheduling, what if someone needs to cancel and reschedule)

Availability: 4 hours

ETA: Finalize plans for emailing and scheduling at Sunday's meeting 08-21-2022. Plan to conduct partner interviews in early-mid September.

**Comment by alice-milmac** [comment]
Progress: Finalized interview template and language for recruitment email. Next steps are to coordinate email and scheduling logistics with PM. 

Blockers: Need to finalize logistics of scheduling interviews (i.e. how my availability will be listed in the calendar, how I will get notified about scheduling, what if someone needs to cancel and reschedule)

Availability: 3 hours

ETA: Finalize plans at Sunday's meeting 09-03-2022. Plan to conduct partner interviews in early-mid September.

**Comment by vaisali89** [comment]
@alice-milmac please add the link to the document with email script under resources.

**Comment by alice-milmac** [comment]
Progress: Added the link for email recruitment text under resources above. 

Blockers: Need to finalize logistics of scheduling interviews (i.e. how my availability will be listed in the calendar, how I will get notified about scheduling, what if someone needs to cancel and reschedule)

Availability: 6 hours

ETA: Finalize plans at Sunday's meeting on 09-11-2022. Plan to conduct partner interviews in mid-September.

**Comment by alice-milmac** [comment]
Progress: All documents associated with this issue have been linked under resources

Blockers: Need to finalize logistics of scheduling interviews (i.e. how my availability will be listed in the calendar, how I will get notified about scheduling, what if someone needs to cancel and reschedule)

Availability: 10 hours

ETA: Finalize plans at Sunday's meeting on 09-11-2022. Plan to conduct partner interviews in mid-September.

**Comment by alice-milmac** [comment]
Progress: Finalized recruitment plan with Bonnie. This week I will:
1 experiment with scheduling though the HfLA research email address to ensure the process works
2 create a spreadsheet with participants information that Bonnie will complete during the next website meeting
3 finalize recruitment email draft including Bonnie's signature to send out during the next website meeting

Blockers: Cannot access 1password information for HfLA research email address. I accepted the invite to 1password and set up an account but cannot see any HfLA information. 

Availability: 10 hours

ETA: Final email recruitment draft to be approved and sent out during the next website meeting

**Comment by alice-milmac** [comment]
Progress: Finalized email recruitment text and process and created spreadsheet for participant contact information. Next steps:
1 establish open Zoom link for interviews that can be added to calendar event text 
2 discuss how to proceed if we don't hear back from any participants (e.g. email goes to spam, how long to wait before contacting again)

Blockers: Final approval for email and Zoom link for interviews

Availability: 15 hours

ETA: Recruiting email to be sent out during Sunday meeting on 09-18-2022

**Comment by alice-milmac** [comment]
Progress: Ran out of time during Sunday's meeting, 09-18-22, for Bonnie to add participant contact information to the spreadsheet. On 09-19-22 I sent Bonnie a request for the information and created an agenda for that request. Resolved issue with Zoom link and can move forward with contacting and scheduling participants. 

Blockers: Need contact information for interview participants 

Availability: 15 hours

ETA: Uncertain but will follow up at next Sunday's meeting if needed.

**Comment by alice-milmac** [comment]
Progress: Added participant information to recruitment spreadsheet and have started emailing participants.

Blockers: None

Availability: 15 hours

ETA: Depends on hearing back from participants and scheduling interviews. Will touch base with the team at our next meeting on Sunday 10-08-2022

**Comment by alice-milmac** [comment]
Progress: Conducted one partner interview and will send follow-up emails to the remaining two this week if they do not schedule. This week I will analyze the interview conducted on 09-30-2022 to uncover emerging themes. 

Blockers: None

Availability: 8 hours

ETA: Depends on hearing back from the remaining two participants. Will touch base with the team at our meeting this Sunday 10-08-2022

**Comment by alice-milmac** [comment]
Progress: Sent follow-up emails to the remaining partner participants (n=2). Analyzed data from first interview and compiled this in the Affinity Mapping doc linked in issue description above. After talking with Bonnie at the Sunday website team meeting the plan moving forward is:
-Email two remaining participants on Wednesday 10-12-2022 and cc Bonnie on the email. If they schedule, I need to update slide in the TDM team meeting presentation for 10-18-2022 [TDM presentation](https://docs.google.com/presentation/d/1l8qEhuv759MAvrXB1am_4jsUuzLyfRA7NDRSPuwB0FY/edit#slide=id.g1640c960b15_0_0)

Blockers: None

Availability: 8 hours

ETA: Depends on hearing back from the remaining two participants. Will touch base with the team at our meeting this Sunday 10-16-2022

**Comment by alice-milmac** [comment]
Progress: Emailed two remaining participants on Wednesday 10-12-2022 and cc'd Bonnie on the email. They hadn't scheduled by 10-18-2022, so the request for them to schedule (and a note for Bonnie to email David about the interview) was included in the TDM team meeting presentation for 10-18-2022 [TDM presentation](https://docs.google.com/presentation/d/1l8qEhuv759MAvrXB1am_4jsUuzLyfRA7NDRSPuwB0FY/edit#slide=id.g1640c960b15_0_0)

Blockers: None

Availability: 8 hours

ETA: Depends on hearing back from the remaining two participants. If the two participants have not scheduled by Friday 10-21-2022 (one full month after initial contact), I think we should close recruitment and proceed with the archetype creation for partners based on my single interview.

**Comment by alice-milmac** [comment]
Progress: Completed all interviews and data analysis and created the following research-based artifacts:

- Partner Persona
- Research Recommendations

Blockers: None

ETA: Research Plan is complete and I will present my findings in a brief presentation during Sunday's website meeting on 10-30-2022

**Comment by ExperimentsInHonesty** [comment]
@alice-milmac you removed your assignment. Generally, issues are closed when the issue is done or the person is leaving off in the middle of the task items, and it needs to be picked up by someone else.

Is there another issue for the presentation?  If there is, please link it here and close this issue.  If this is the only issue, then please keep this open and yourself assigned until you link the presentation in a comment here.  After you have given the presentation, you can close the issue.  

If there is something I am not understanding, please provide more details.

**Comment by alice-milmac** [comment]
Hi @ExperimentsInHonesty I'm sorry for any confusion I have caused. I believe this issue is completely resolved and should be closed. I linked the presentation above as the last item under "Created Assets" in the Overview above and I'm also linking [the presentation here](https://docs.google.com/presentation/d/1b4VXYcDgIno90wPNw6vTv2qj-mRDW_0lfCDiHX73WCM/edit#slide=id.g178e899cf39_0_37).

I will also close this issue after this comment. Again, apologies for any confusion I caused in navigating the conclusion of the project and process in Github.
